packages/tailwind/docs/styles.module.scss

```diff
@@ -1,3 +1,7 @@
+// Would normally be imported in project's main css file.
+// Needed for shadow styles etc.
+@tailwind base;
```

Comment:
Just wondering, this is not the TW CSS reset, is it? I think we made a decision that we're not endorsing it for projects in favor of the already baked in reset.
So if we were using it in our own docs, that might mean we would want to revisit that decision?

Reply:
No the CSS reset is called [Preflight](https://tailwindcss.com/docs/preflight). If you look at the css `base` compiles to, it's basically just a bunch of `--tw` prefixed css variables.
The docs [here](https://tailwindcss.com/docs/upgrade-guide#base-layer-must-be-present) also mention that it's a requirement from `3.0` onwards

Reply:
Yeah, this caught us out for a while when we were exploring focus rings. It kind of silently does nothing with no real errors to let the user. We should add a line into the docs just to call that out :)